INC-5312 fix missing image by using a default image

alfasin opened the pull request:
# Description

Fix image bug: we should display a default placeholder instead of a missing image

<img width="1403" alt="image" src="https://user-images.githubusercontent.com/1937869/179569002-63afd41c-e772-4008-8024-51af1209adfc.png">

vanessa-cooper committed:
Fixing a Bug

vanessa-cooper reviewed (changes requested):
Make sure all checks pass, only then you can merge the PR.

alfasin committed:
INC-5312

Fix image bug: we should display a default placeholder instead of a missing image

alfasin committed:
INC-5312

Fix image bug: we should display a default placeholder instead of a missing image
